api/db-cdr/generate-cdr/README.md

```diff
@@ -0,0 +1,37 @@
+COUNT GENERATION
+
+This document describes what counts are generated for each of the data domains
+
+Total Person count is generated and stored with analysis id 1.
+Person count by gender is generated and stored with analysis id 2.
+Person count by year of birth is generated and stored with analysis id 3.
+Person count by race is generated and stored with analysis id 4.
+Person count by ethnicity is generated and stored with analysis id 5.
+Person count by year of birth and gender is generated and stored with analysis id id 10.
```

Comment:
id id ... (too many id)

Reply:
Formatting of this section in readme needs work. Use markdown like in other readmes.